localScrapper.js

```diff
@@ -77,10 +82,23 @@ function createDataDir() {
 const getFormattedContent = () => new Promise((resolve, reject) => {
   getDirectoryContents('./karaoke', (err, contents) => {
     if (err) return reject(err)
-    const allContents = contents
+    const subtitlesContentsMap = contents
+      .filter(content => content.isSubtitles)
+      .reduce(
+        (prev, curr) => Object.assign({}, prev, { [curr.pathWithoutExtension]: curr.path }), {}
```

Comment:
Oh I didn't know about it, thanks :)
However, with groupBy I'd get an object containing multiples entries with
`{ [content.pathWithoutExtension]: content }`
when I want `{ [content.pathWithoutExtension]: content.path }`